### PR TITLE
[system-health] [update] Add fan direction check to system health

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
**What I did?**
Add a fan direction check to system health.

**Why I did this?**
Correct airflow is very important for a switch system to prevent it from overheating. Incorrect airflow would cause abnormal system temperature raising and it is difficult to debug it without open the switch box. 

As fan direction is already queried and stored to DB by thermalctld, system-health service should monitor fan direction and raise alarm if fan direction is not correct. 

**Related PRs**

| PR title | state | context |
| ---------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------ |
| [[system-health] Add fan direction check for system health ](https://github.com/sonic-net/sonic-buildimage/pull/14509) | GitHub issue/pull request detail | GitHub pull request check contexts |
| [[system-health] Add fan dir test for system health test ](https://github.com/sonic-net/sonic-mgmt/pull/8100) | GitHub issue/pull request detail | GitHub pull request check contexts |

### CODE REVIEW

**Comment by liat-grozovik** [comment]
@prgeor @keboliu please review

**Review by keboliu — APPROVED** [review]

File: dummy
Comment (prgeor) [inline dummy:1]: @Junchao-Mellanox what is this dummy file?

File: doc/system_health_monitoring/system-health-HLD.md [hunk 1]
```diff
@@ -55,6 +55,9 @@ By default any service is not in expected status will be considered as fault con
 
 -  Any fan is missing/broken
 -  Fan speed is lower than minimal value
+-  Fan direction is not expected
+    - All fan direction must be the same
```
Comment (prgeor) [inline doc/system_health_monitoring/system-health-HLD.md:59]: same direction for ALL fans across all planforms?

Comment (Junchao-Mellanox) [reply doc/system_health_monitoring/system-health-HLD.md:59]: Same direction for the same platform

File: dummy
Comment (Junchao-Mellanox) [reply dummy:1]: I will remove it

Comment (Junchao-Mellanox) [reply dummy:1]: Removed

**Review by liat-grozovik — APPROVED** [review]

**Comment by liat-grozovik** [comment]
@prgeor any further feedback or we can merge and focus on code PRs?

**Review by prgeor — APPROVED** [review]

**Pull request merged**